src/content/en/fundamentals/discovery-and-monetization/payment-request/deep-dive-into-payment-request.md

```diff
@@ -52,9 +52,12 @@ two major Chrome versions.
 
 ## Feature Detect
 
-Before we start covering how to use PaymentRequest, we should feature detect to
-ensure it's available and fallback to a traditional checkout page in browsers
-that don't support it. The feature detect is simply:
+PaymentRequest has been available since Chrome 53 for Android and has been
```

Comment:
> Please do this throughout *your* article.

While this is @gauntface's article, I've [made](https://github.com/google/WebFundamentals/pull/4703/files#diff-c2eeb64631b87bab660657a09692140c) the requested changes.

There's another kind of occurrence, "Payment Request UI", which was also inconsistent (e.g. `payment request UI` on line 1630. I've opted for the latter.